src/store/sagas/wallet.ts

```diff
@@ -178,35 +199,43 @@ export function* signAndSend(wallet: WalletAdapter, tx: Transaction): SagaGenera
   const signature = yield* call([connection, connection.sendRawTransaction], signedTx.serialize())
   return signature
 }
-export function* createAccount(tokenAddress: PublicKey): SagaGenerator<PublicKey> {
+export function* createAccount(tokenAddress: PublicKey[]): SagaGenerator<PublicKey[]> {
```

Comment:
we're using this saga also in swap and positions, so there might be types errors

Reply:
done, i created second function to create multiple accounts